Emby.Server.Implementations/Data/SqliteUserRepository.cs

```diff
@@ -180,14 +179,10 @@ private User GetUser(IReadOnlyList<IResultSetValue> row)
             var id = row[0].ToInt64();
             var guid = row[1].ReadGuidFromBlob();
```

Comment:
Should we remove this extension as well? Or is it only the deserialization that sucks?

Reply:
It’s just the Servicestack deserialization from raw utf8 that sucks. This extension method just calls the Guid ctor (and copies the byte span to a new array).